_unused/lab-swedlow.html

```diff
@@ -5,7 +5,7 @@
         <div class="callout large primary">
             <div class="row column text-center">
                 <h1>Swedlow Lab</h1>
-                <p>Prof Jason Swedlow, Wellcome Trust Centre for Gene Regulation &amp; Expression, University of Dundee, UK</p>
+                <p>Prof Jason Swedlow, Discovery Centre for Translational &amp; Interdisciplinary Research, University of Dundee, UK</p>
```

Comment:
Although the Discover Centre is the correct name for our building, we should rather use 

```
Division of Computational Biology, School of Life Sciences, University of Dundee
```

for the affiliation across the board.